### PR TITLE
Feature/fallback labels

### PR DESCRIPTION
**Pull request by robin-homberg**

### CODE REVIEW

File: Model/Fallback.php [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+        if (count($route) === 0) {
+            $route[] = $column;
+        }
+
+        return $route;
+    }
+
+    /**
+     * Get SQL Case when query for fallback route
+     *
```
Comment (robin-homberg) [inline Model/Fallback.php:233]: wrong phpdoc

File: Model/Fallback.php [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+            $route[] = $column;
+
+            if (is_string($this->fallbackRoute[$column]['fallback'])) {
+                $column = $this->fallbackRoute[$column]['fallback'];
+                continue;
+            }
+
+            break;
+
+            $try++;
```
Comment (BrocksiNet) [inline Model/Fallback.php:216]: Unreachable code

File: Job/Attribute.php [hunk 3]
```diff
@@ -338,9 +356,12 @@ public function addAttributes()
 
             /* Retrieve default admin label */
             /** @var string $frontendLabel */
-            $frontendLabel = __('Unknown');
-            if (!empty($row[$adminLabelColumn])) {
-                $frontendLabel = $row[$adminLabelColumn];
+            $frontendLabel = __('Unkown');
```
Comment (BrocksiNet) [inline Job/Attribute.php]: typo

File: Job/Category.php [hunk 4]
```diff
@@ -472,20 +496,34 @@ public function setValues()
...
         /**
          * @var string $local
          * @var array $affected
          */
         foreach ($stores as $local => $affected) {
-            if (!$connection->tableColumnExists($tmpTable, 'labels-' . $local)) {
+            $labelColumn = 'labels-' . $local;
+
+            $sqlCase = sprintf('`%s`', $labelColumn);
+            if ($this->configHelper->useLabelFallbacks()) {
```
Comment (BrocksiNet) [inline Job/Category.php]: DRY

**Review by BrocksiNet — APPROVED** [review]

**Pull request merged**